roles/openafs_krbserver/templates/krb5.conf.j2

```diff
@@ -2,6 +2,7 @@
 
 [libdefaults]
     default_realm = {{ afs_realm }}
+    dns_lookup_kdc = false
```

Comment:
Yes, but in some cases we want to lookup the kdc with dns. For example when running on linux on real cells, we use srv records to lookup the kdcs. Perhaps this should be a platform specific value?  Or am I mis-understanding?